package.json

```diff
@@ -42,6 +42,9 @@
     "typescript": "^4.6.4"
   },
   "dependencies": {
+    "@bonfida/name-offers": "^0.0.1",
+    "@bonfida/spl-name-service": "^0.1.37",
+    "@cardinal/namespaces": "^4.1.7",
```

Comment:
I don't think it's a good approach to add libraries directly to sdk, cause currently we have issue with bundle size due to this libraries.
I'd suggest a modular approach, and split it to different packages like `@dialectlabs/name-service-cardinal` or something like this. And have them dynamically added and configured to the integration where it'll be needed